lib/tests/streamlit/elements/heading_test.py

```diff
@@ -162,3 +223,11 @@ def test_st_title_with_help(self):
         self.assertEqual(el.heading.body, "some title")
         self.assertEqual(el.heading.tag, "h1")
         self.assertEqual(el.heading.help, "help text")
+        self.assertFalse(el.heading.divider)
+
+    def test_st_title_with_invalid_divider(self):
+        """Test st.title with invalid divider."""
```

Comment:
nit: `st.title` doesn't support divider at all or? Maybe rename the comment to: Test st.title doesn't support divider parameter.

Reply:
Correct, they only wanted divider option on `st.header` & `st.subheader`